packages/clerk-js/package.json

```diff
@@ -31,24 +31,24 @@
   ],
   "scripts": {
     "build": "npm run build:bundle && npm run build:declarations",
-    "build:analyze": "webpack-bundle-analyzer stats.json dist/",
-    "build:bundle": "npm run clean && webpack --config webpack.config.js --env production",
+    "build:analyze": "rspack build --config rspack.config.js --env production --env variant=\"clerk.browser\" --env analysis --analyze",
+    "build:bundle": "npm run clean && rspack build --config rspack.config.js --env production",
     "build:declarations": "tsc -p tsconfig.declarations.json",
-    "build:stats": "webpack --config webpack.config.js --env production --json=stats.json --env variant=\"clerk.browser\"",
+    "build:stats": "rspack build --config rspack.config.js --env production --json=stats.json --env variant=\"clerk.browser\"",
     "bundlewatch": "npx bundlewatch --config bundlewatch.config.json",
     "clean": "rimraf ./dist",
-    "dev": "webpack serve --config webpack.config.js",
-    "dev:headless": "webpack serve --config webpack.config.js --env variant=\"clerk.headless.browser\"",
+    "dev": "rspack serve --config rspack.config.js",
+    "dev:headless": "rspack serve --config rspack.config.js --env variant=\"clerk.headless.browser\"",
     "lint": "eslint src/",
     "lint:attw": "attw --pack .",
     "lint:publint": "publint || true",
     "test": "jest",
     "test:cache:clear": "jest --clearCache --useStderr",
     "test:ci": "jest --maxWorkers=70%",
     "test:coverage": "jest --collectCoverage && open coverage/lcov-report/index.html",
-    "watch": "webpack --config webpack.config.js --env production --watch"
+    "watch": "rspack build --config rspack.config.js --env production --watch"
   },
-  "browserslist": "last 2 versions, ios_saf > 12, Safari > 12, > 1%, not dead, not ie > 0",
+  "browserslist": "last 2 years",
```

Comment:
This is essentially Chrome >108, Safari >16.2, and Firefox >107.

Reply:
Don't we still need `ios_saf > 12, Safari > 12` ?

Reply:
We have one customer who was using v4 on versions as old as Safari 12. After discussing it with the team I believe we're going to implement a specific build for that customer and serve it via the Cloudflare Worker.

Prior to this PR, this `browserslist` config had no impact on the final build. I confirmed by setting it to `Safari 9` (which did not support arrow functions `() => {}`) and confirmed that the resulting JavaScript files still contained arrow functions.

Ultimately, I think this meant that our older browser support wasn't tool enforced; it was solely a result of the APIs and JS language features we used.

Reply:
I think, for now, I'm going to set the `browserslist` property to `last 2 years, Safari > 12, iOS > 12`. We can explore dropping the Safari support in a future PR once we've implemented custom builds for customers who need it.